interfaces/system_key.go

```diff
@@ -39,6 +39,7 @@ type systemKey struct {
 	BuildID          string   `yaml:"build-id"`
 	AppArmorFeatures []string `yaml:"apparmor-features"`
 	NFSHome          bool     `yaml:"nfs-home"`
+	OverlayRoot      bool     `yaml:"overlay-root"`
```

Comment:
Could we use a string here? An empty string would mean no overlay and any other value would be the upper_dir we want to use anyway.

Reply:
Yes, this is how I would change it to capture the parsed value of upperdir.

Reply:
Done